R/endpoint.R

```diff
@@ -166,6 +166,7 @@ porcelain_endpoint <- R6::R6Class(
     ##' @param ... Additional arguments passed through to \code{run}
     plumber = function(req, res, ...) {
       tryCatch({
+        req$endpoint <- self$path
```

Comment:
```suggestion
        req$porcelain_endpoint <- self$path
```

perhaps? that we use that I think elsewhere to avoid any possible collisions with plumber and httpuv

Reply:
Yeah sounds sensible